src/PostgresOutbox/PostgresOutbox.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net8.0</TargetFramework>
+    </PropertyGroup>
+    
+    <ItemGroup>
+        <PackageReference Include="JetBrains.Annotations" Version="2023.3.0" />
```

Comment:
**QUESTION:** What's the benefit of adding this dependency?

Reply:
You can mark it as development dependency
https://learn.microsoft.com/en-us/nuget/consume-packages/package-references-in-project-files#controlling-dependency-assets

Reply:
Added issue for that: https://github.com/oskardudycz/PostgresOutboxPatternWithCDC.NET/issues/10